raster/r.surf.gauss/testsuite/test_surf_gauss.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+"""
+MODULE:    Test of r.surf.gauss
+
+AUTHOR(S): Corey White <ctwhite48 gmail com>
+
+PURPOSE: Tests random gauss surface generation
+
+COPYRIGHT: (C) 2023 - 2024 by Corey White and the GRASS Development Team
+
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+"""
+
+import os
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class MeanSigmaTestCase(TestCase):
+    """Test min and max of r.surf.random module"""
+
+    # Raster map name be used as output
+    output = "random_result"
+
+    @classmethod
+    def setUpClass(cls):
+        """Ensures expected computational region"""
+        os.environ["GRASS_RANDOM_SEED"] = "42"
+        # modifying region just for this script
+        cls.use_temp_region()
+        # Only 100,000,000 seem to resonably (not 100%) ensure that all values
+        # are generated, so exceeding of ranges actually shows up.
+        cls.runModule("g.region", rows=10000, cols=10000)
```

Comment:
The test runs 46 s for me, that's unnecessarily long...

Reply:
I reduced the extent to 1000X1000 and it now runs in about a second.

Reply:
What's the smallest size that tests the same things? Is 5x12 or other enough?

Reply:
I'm not sure what the appropriate spatial scale is to capture all of the variability in the Gaussian random field. I did some basic tests that fail if we go lower than 1000x1000 unless we lower the precision when checking the univariate stats.

The other option is to just test if the seed options are getting set correctly for this PR.

Reply:
I changed the tests to just check that the r.univar values match. The seed variable was anyway set to 42, so the results were always the same even if the seed option was not used.